src/docs/README.md

```diff
@@ -41,18 +41,34 @@ For a general introduction to writing markdown, see: [Markdown tutorial](tutoria
    
   ![Git Routine](./img/contribution_git.png)
 
-3. You can work and make any changes on your own fork.
+3. You can work and make any changes in your own repository and commit + push them to your fork.
 
 4. Once you want to submit those changes to the main repository, you can open a "pull request" by clicking "Contribute" (5) in the top-right corner.
    > Remember to "Fetch upstream" (4), if your fork is not up-to-date with the main repository.
 
 5. If edited or added existing content, please assign the original author during your pull-request to review your changes.
+  
   - The github user name of the original author is stored in the yaml block on top under `author_github`
   - During the pull-request mention the author via `@<author_github>`.
 
-# (Live) Testing
+## Testing
```

Comment:
Update toc if you change header level here

Reply:
TOC is already removed / added to yaml.

Reply:
Actually please revert to manual toc. I know this is cumbersome, but the nfdi-toc is not generated in GitHub.

Reply:
Ah good point.